drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/report.rb

```diff
@@ -338,10 +417,53 @@ def overall_current_living_situation_count
       @overall_current_living_situation_count ||= CurrentLivingSituation.current_living_situation_scope(self).count
     end
 
-    private def percent(total, partial)
+    def percent(total, partial)
       return 0 if total.blank? || total.zero? || partial.blank? || partial.zero?
 
-      ((partial / total.to_f) * 100).round
+      ((partial / total.to_f) * 100).round(1)
+    end
+
+    private def client_ids_for_project(project)
+      @client_ids_per_project ||= enrollments.pluck(:project_id, :destination_client_id).group_by(&:shift)
+      @client_ids_per_project[project.id]&.flatten
+    end
+
+    private def count_category_setup
+      setup = { 'Overall' => { invalid_count: 0, total: 0 } }
+      categories.each do |cat|
+        setup[cat] = { invalid_count: 0, total: 0 }
+      end
+      setup
```

Comment:
You added 'Overall' to `categories` above, so:
```suggestion
      {}.tap do |setup|
      categories.each do |cat|
        setup[cat] = { invalid_count: 0, total: 0 }
      end
```

Reply:
Whoops!